src/jmclient/storage.py

```diff
@@ -25,16 +38,17 @@ def __init__(self, password, salt=None, hash_len=32, salt_len=16,
         # Default and recommended settings from argon2.PasswordHasher are for
         # interactive logins. For encryption we want something much slower.
         self.settings = {
-            'time_cost': time_cost,
-            'memory_cost': memory_cost,
-            'parallelism': parallelism,
-            'hash_len': hash_len,
-            'type': argon2_type,
-            'version': version
+            "time_cost": time_cost,
```

Comment:
Can these style-only unrelated changes be omitted from this PR?

Reply:
I have removed all quote formatting changes, but left in standard "new line" formatting changes. It feels obvious to me to let tools do their formatting, but i'll be happy to remove these too if needed.

Reply:
thinking about it, i reverted style only changes. which i will address in style-only related pr